src/test/Test.RazorEngine.Core/IsolatedRazorEngineServiceTestFixture.cs

```diff
@@ -685,11 +688,11 @@ public void IsolatedRazorEngineService_CheckThatWeCanUseUnknownTypesAtExecuteTim
     var t = new TestHelper.TestClass();
 }
 @t.TestProperty";
-                const string expected = "\n\nTestPropert";
+                const string expected = "TestPropert";
```

Comment:
With `RAZOR4` the expected result is `\nTestPropert`, so maybe I should add an additional `#if RAZOR4` compiler directive instead of checking with `Contains`.